CHANGELOG.md

```diff
@@ -118,6 +122,8 @@ These functions and structs can be referenced in both code and blueprints it may
 - Fixed a rare crash where a RepNotify callback can modify a GDK data structure being iterated upon.
 - Fixed race condition in Spatial Test framework that would cause tests to time out with one or more workers not ready to begin the test.
 - Fixed client connection not being cleaned up when moving out of interest of a server.
+- Fixed a assertion being triggered on async loaded entities due to queuing some component addition.
```

Comment:
Are these new changelog entries correct? New bugfixes in the rc being brought over into master?

Reply:
This is from Nics PR into RC: https://github.com/spatialos/UnrealGDK/pull/2770